docs/types/schema.md

```diff
@@ -45,45 +45,39 @@ Creates a GraphQL schema
 - `mutation`: The root mutation type
 - `subscription`: The root subscription type
 
-```python
-async Schema().execute(
-    query: str,
-    variable_values: Optional[Dict[str, Any]] = None,
-    context_value: Optional[Any] = None,
-    root_value: Optional[Any] = None,
-    operation_name: Optional[str] = None,
-    validate_queries: Optional[bool] = True,
-) -> ExecutionResult:
-```
+### `.execute()` (async)
 
-Executes a GraphQL operation:
+Executes a GraphQL operation against a schema (async)
 
-- `query`: The document to be executed
-- `variable_values`: The variables for this operation
-- `context_value`: The value of the context that will be passed down to
-  resolvers
-- `root_value`: The value for the root type that will passed down to root
-  resolvers
-- `operation_name`: The name of the operation you want to execute, useful when
-  sending a document with multiple operations
-- `validate_queries`: This flag allows to disable query validation
+`execute(query, variable_values, context_value, root_value, operation_name, validate_queries)`
 
-```python
-Schema().execute_sync(
-    query: str,
-    variable_values: Optional[Dict[str, Any]] = None,
-    context_value: Optional[Any] = None,
-    root_value: Optional[Any] = None,
-    operation_name: Optional[str] = None,
-    validate_queries: Optional[bool] = True,
-) -> ExecutionResult:
-```
+| Parameter name   | Type                              | Description                                                                                            |
+| ---------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------ |
```

Comment:
Maybe it would be better to have a column for defaults instead of bundling it with Type?

Reply:
great idea!